ExtensionPack/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# C/C++ Extension Pack for Visual Studio Code Change Log
+
+
+## Version 0.1.0: September 1, 2020
+### List of extensions:
+* C/C++ (Microsoft)
+* C/C++ Themes (Microsoft)
+* Cmake Tools (Microsoft)
+* Remote Development extension pack (Microsoft)
+* GitHub Pull Requests and Issues (GitHub)
+* Visual Studio Codespaces (Microsoft)
```

Comment:
Do the VS Codespace and LiveShare extensions work well with our extension? Has anyone tried?

Reply:
@jureid have you tried them together?

Reply:
Yes I have tried both, they do

Reply:
Also remove (Microsoft) and (GitHub) from here
Capitalize "Extension Pack"